.github/workflows/mac.yml

```diff
@@ -45,12 +45,10 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: ${{ matrix.python-version }}
-          architecture: x64
 
       - name: setup
         run: |
           pip3 install pytest tox
-          #brew install valgrind # TODO: does not work
           mkdir builds
 
       - name: download-planner
```

Comment:
I'd use `tox -e py,search` below. This will run the exitcodes test and run the standard configs, which both have some OS-dependent code. I don't think we need to test the Python translator code on macOS.

Reply:
Does the translator include the py test? If yes, I don't think it hurts to run the translator tests here, too.They are fast, I tihnk.

Reply:
The "py" tests call the "testenv" section of tox, so in our case:

```
  pytest driver/tests.py misc/tests/test-exitcodes.py
  bash -c "python3 misc/autodoc/autodoc.py --dry-run > /dev/null"
```

I'm fine with running the translator tests on macOS, so "tox -e py,translator,search".

Reply:
The py tests fail because we don't have validate installed and don't know how to do so. Unless you have a better idea, I will stick with translator and search tests.

Reply:
Ah, thanks! I think it would be good to add this as a comment.